cmd/kube-proxy/app/server.go

```diff
@@ -59,8 +56,6 @@ func NewProxyServer() *ProxyServer {
 
 // AddFlags adds flags for a specific ProxyServer to the specified FlagSet
 func (s *ProxyServer) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&s.EtcdConfigFile, "etcd_config", s.EtcdConfigFile, "The config file for the etcd client. Mutually exclusive with -etcd_servers")
```

Comment:
@thockin + @roberthbailey perhaps calling out these changes in the changelog for the future.  So folks will be aware of sea-changes that will break their updates.  

It's how we roll errata's. 
@eparis ^ FYI.